Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 16
Rules claiming more threads will be scaled down.
Job counts:
	count	jobs
	4	call_cpg
	4	mbed_to_mfreq
	4	mtsv_to_mbed
	4	nanopolish_index
	1	parse_nanopore
	17

[Sun Mar 10 11:04:33 2019]
rule nanopolish_index:
    input: data/nanopore/reads/CHOZNStableNoglnDay0rep2.fastq.gz
    output: data/nanopore/reads/CHOZNStableNoglnDay0rep2.fastq.gz.index.readdb
    jobid: 17
    wildcards: pre=data/nanopore/reads/CHOZNStableNoglnDay0rep2


[Sun Mar 10 11:04:33 2019]
rule nanopolish_index:
    input: data/nanopore/reads/CHOZNStableNoglnDay0rep3.fastq.gz
    output: data/nanopore/reads/CHOZNStableNoglnDay0rep3.fastq.gz.index.readdb
    jobid: 15
    wildcards: pre=data/nanopore/reads/CHOZNStableNoglnDay0rep3


[Sun Mar 10 11:04:33 2019]
rule nanopolish_index:
    input: data/nanopore/reads/CHOZNUnstableNoglnDay0rep2.fastq.gz
    output: data/nanopore/reads/CHOZNUnstableNoglnDay0rep2.fastq.gz.index.readdb
    jobid: 13
    wildcards: pre=data/nanopore/reads/CHOZNUnstableNoglnDay0rep2


[Sun Mar 10 11:04:33 2019]
rule nanopolish_index:
    input: data/nanopore/reads/CHOZNUnstableNoglnDay0rep3.fastq.gz
    output: data/nanopore/reads/CHOZNUnstableNoglnDay0rep3.fastq.gz.index.readdb
    jobid: 20
    wildcards: pre=data/nanopore/reads/CHOZNUnstableNoglnDay0rep3

[Sun Mar 10 11:04:33 2019]
Error in rule nanopolish_index:
    jobid: 17
    output: data/nanopore/reads/CHOZNStableNoglnDay0rep2.fastq.gz.index.readdb

RuleException:
CalledProcessError in line 29 of /home/isac/Code/projects/ambic-epigenome/snakemake/nanopore_data_parse.smk:
Command ' set -euo pipefail;  nanopolish index --verbose -d data/nanopore/reads/CHOZNStableNoglnDay0rep2 -s data/nanopore/reads/CHOZNStableNoglnDay0rep2.summary.txt data/nanopore/reads/CHOZNStableNoglnDay0rep2.fastq.gz &> data/nanopore/reads/CHOZNStableNoglnDay0rep2.index.log ' returned non-zero exit status 1.
  File "/home/isac/Code/projects/ambic-epigenome/snakemake/nanopore_data_parse.smk", line 29, in __rule_nanopolish_index
  File "/home/isac/.conda/envs/isacenv/lib/python3.6/concurrent/futures/thread.py", line 56, in run
[Sun Mar 10 11:04:33 2019]
Error in rule nanopolish_index:
    jobid: 15
    output: data/nanopore/reads/CHOZNStableNoglnDay0rep3.fastq.gz.index.readdb

RuleException:
CalledProcessError in line 29 of /home/isac/Code/projects/ambic-epigenome/snakemake/nanopore_data_parse.smk:
Command ' set -euo pipefail;  nanopolish index --verbose -d data/nanopore/reads/CHOZNStableNoglnDay0rep3 -s data/nanopore/reads/CHOZNStableNoglnDay0rep3.summary.txt data/nanopore/reads/CHOZNStableNoglnDay0rep3.fastq.gz &> data/nanopore/reads/CHOZNStableNoglnDay0rep3.index.log ' returned non-zero exit status 1.
  File "/home/isac/Code/projects/ambic-epigenome/snakemake/nanopore_data_parse.smk", line 29, in __rule_nanopolish_index
  File "/home/isac/.conda/envs/isacenv/lib/python3.6/concurrent/futures/thread.py", line 56, in run
[Sun Mar 10 11:04:33 2019]
Error in rule nanopolish_index:
    jobid: 13
    output: data/nanopore/reads/CHOZNUnstableNoglnDay0rep2.fastq.gz.index.readdb

RuleException:
CalledProcessError in line 29 of /home/isac/Code/projects/ambic-epigenome/snakemake/nanopore_data_parse.smk:
Command ' set -euo pipefail;  nanopolish index --verbose -d data/nanopore/reads/CHOZNUnstableNoglnDay0rep2 -s data/nanopore/reads/CHOZNUnstableNoglnDay0rep2.summary.txt data/nanopore/reads/CHOZNUnstableNoglnDay0rep2.fastq.gz &> data/nanopore/reads/CHOZNUnstableNoglnDay0rep2.index.log ' returned non-zero exit status 1.
  File "/home/isac/Code/projects/ambic-epigenome/snakemake/nanopore_data_parse.smk", line 29, in __rule_nanopolish_index
  File "/home/isac/.conda/envs/isacenv/lib/python3.6/concurrent/futures/thread.py", line 56, in run
[Sun Mar 10 11:04:33 2019]
Error in rule nanopolish_index:
    jobid: 20
    output: data/nanopore/reads/CHOZNUnstableNoglnDay0rep3.fastq.gz.index.readdb

RuleException:
CalledProcessError in line 29 of /home/isac/Code/projects/ambic-epigenome/snakemake/nanopore_data_parse.smk:
Command ' set -euo pipefail;  nanopolish index --verbose -d data/nanopore/reads/CHOZNUnstableNoglnDay0rep3 -s data/nanopore/reads/CHOZNUnstableNoglnDay0rep3.summary.txt data/nanopore/reads/CHOZNUnstableNoglnDay0rep3.fastq.gz &> data/nanopore/reads/CHOZNUnstableNoglnDay0rep3.index.log ' returned non-zero exit status 1.
  File "/home/isac/Code/projects/ambic-epigenome/snakemake/nanopore_data_parse.smk", line 29, in __rule_nanopolish_index
  File "/home/isac/.conda/envs/isacenv/lib/python3.6/concurrent/futures/thread.py", line 56, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /home/isac/Code/projects/ambic-epigenome/.snakemake/log/2019-03-10T110432.830280.snakemake.log
